components/Elevation/src/UIView+MaterialElevationResponding.h

```diff
@@ -33,4 +33,12 @@
  */
 @property(nonatomic, assign, readonly) CGFloat mdc_baseElevation;
 
+/**
+ Should be called when the view's elevation has changed. Will be called on the reciever's @c
```

Comment:
This should be called when the component's current elevation changes ( mdc_currentElevation ), concretely this is can be triggered either when the client sets the elevation of the component (for the state it is currently in), or the elevation changes because there is a tap or interaction (which causes a state change).

but if it's not clear then could be worth documenting.

Reply:
Updated the docs.